scripts/build.py

```
@@ -14,6 +14,8 @@
 logging.basicConfig(level=logging.INFO)
 
 
+
```

Did you add these to make sure it fails fast?

---

yes

---

Will remove before submitting (Assuming it fails fast)

---

I think we may need to change some travis setting to make it use the travis.yml file in the PR instead of the one in master

---

no, it will pick up changes in the new branch. i think the indent levels are off. at the top level, it goes stage, stage, if, if, ...